### PR TITLE
added draw pet stats

### PR DESCRIPTION
**Pull request by Zodey-hub**

### CODE REVIEW

**Review by Pablete1234 — CHANGES_REQUESTED** [review]

File: src/main/java/com/github/manolo8/darkbot/config/ColorScheme.java [hunk 1]
```diff
@@ -15,6 +15,7 @@
     public @Option Color PORTALS = new Color(0xAEAEAE);
     public @Option Color HERO = new Color(0x22CC22);
     public @Option @Col(alpha = false) Color TRAIL_BASE = new Color(0xE0E0E0);
+    public @Option Color FUEL = new Color(0xF2F2F2);
```
Comment (Pablete1234) [inline src/main/java/com/github/manolo8/darkbot/config/ColorScheme.java:18]: This is missing a the translation key `cs.fuel` in the strings_en file

**Review by Pablete1234 — APPROVED** [review]

**Comment by Zodey-hub** [comment]
Do I need to test it with OBS then send it?

**Comment by Pablete1234** [comment]
If you want to record a bit of how it's running etc that'd be nice, i still want to checkout the code locally and run it for a bit to double check everything runs smoothly

**Comment by Zodey-hub** [comment]
Okey, 5 min and I send it

**Comment by Zodey-hub** [comment]
https://streamable.com/7kd09z

**Comment by Pablete1234** [comment]
From what i see, while loading pet health bar is shown, that's a bug.

The constructor for Pet that has no parameters should set `removed = true;` which will be updated later when addresses are found etc

**Pull request merged**